feat(eslint-plugin-fiori-tools): lint flex change files (#4706)
SAP/open-ux-tools: packages/eslint-plugin-fiori-tools/README.md

|   9.8.0   | [sap-table-personalization](docs/rules/sap-table-personalization.md) | Ensures that all table `personalization` options are enabled in the OData V4 applications. | | ✅ |
|   9.7.0   | [sap-anchor-bar-visible](docs/rules/sap-anchor-bar-visible.md) | Anchor Bar Visible should not be set to false in manifest settings for object page headers (except form entry object pages). | | ✅ |
|   9.5.0   | [sap-table-column-vertical-alignment](docs/rules/sap-table-column-vertical-alignment.md) | Ensures `tableColumnVerticalAlignment` Configuration for Responsive Type Tables in SAP Fiori Elements applications | | ✅ |

> |   9.4.0   | [sap-enable-export](docs/rules/sap-enable-export.md) | Ensures that the export to Excel functionality in any OData V4 and OData V2 applications tables is available | | ✅ |

Reviewer: ```suggestion
|   9.4.0   | [sap-enable-export](docs/rules/sap-enable-export.md) | Ensures that the export to Excel functionality for tables in OData V4 and OData V2 applications is enabled | | ✅ |
```

> |   9.4.0   | [sap-enable-paste](docs/rules/sap-enable-paste.md) | Ensures that the paste functionality in any OData V4 and OData V2 applications tables is available | | ✅ |

Reviewer: ```suggestion
|   9.4.0   | [sap-enable-paste](docs/rules/sap-enable-paste.md) | Ensures that the paste functionality for tables in OData V4 and OData V2 applications tables is enabled | | ✅ |
```
